ui/lib/src/view/verticalResize.ts

```diff
@@ -12,56 +11,62 @@ interface Opts {
-          el.style.height = `${clamp(height, { min: o.min(), max: o.max() })}px`;
-        };
-        onDomChange();
+      hook: {
+        insert: vn => {
+          const divider = vn.elm as ResizerElement;
+          const onDomChange = () => {
+            const el = divider.previousElementSibling as HTMLElement;
+            if (el.style.height) return;
+            let height = o.id && heightStore(`${o.key}.${o.id}`);
+            if (typeof height !== 'number') height = heightStore(o.key) ?? o.initialMaxHeight;
+            if (typeof height !== 'number') height = el.getBoundingClientRect().height;
+            el.style.flex = 'none';
+            el.style.height = `${clamp(height, { min: o.min(), max: o.max() })}px`;
+          };
+          onDomChange();
 
-        new MutationObserver(onDomChange).observe(divider.parentElement!, { childList: true });
+          divider.observer = new MutationObserver(onDomChange);
+          divider.observer.observe(divider.parentElement!, { childList: true });
 
-        divider.addEventListener('pointerdown', down => {
-          const el = divider.previousElementSibling as HTMLElement;
-          const beginFrom = el.getBoundingClientRect().height - down.clientY;
-          divider.setPointerCapture(down.pointerId);
+          divider.addEventListener('pointerdown', down => {
+            const el = divider.previousElementSibling as HTMLElement;
+            const beginFrom = el.getBoundingClientRect().height - down.clientY;
+            divider.setPointerCapture(down.pointerId);
 
-          const move = (move: PointerEvent) => {
-            el.style.height = `${clamp(beginFrom + move.clientY, { min: o.min(), max: o.max() })}px`;
-          };
+            const move = (move: PointerEvent) => {
+              el.style.height = `${clamp(beginFrom + move.clientY, { min: o.min(), max: o.max() })}px`;
+            };
 
-          const up = () => {
-            divider.releasePointerCapture(down.pointerId);
-            window.removeEventListener('pointermove', move);
-            window.removeEventListener('pointerup', up);
-            window.removeEventListener('pointercancel', up);
-            const height = parseInt(el.style.height);
-            heightStore(o.key, height);
-            if (o.id) heightStore(`${o.key}.${o.id}`, height);
-          };
-          window.addEventListener('pointermove', move);
-          window.addEventListener('pointerup', up);
-          window.addEventListener('pointercancel', up);
-        });
-      }),
+            const up = () => {
+              divider.releasePointerCapture(down.pointerId);
+              window.removeEventListener('pointermove', move);
+              window.removeEventListener('pointerup', up);
+              window.removeEventListener('pointercancel', up);
+              const height = parseInt(el.style.height);
+              heightStore(o.key, height);
+              if (o.id) heightStore(`${o.key}.${o.id}`, height);
```

Comment:
this is a default/global stored height mechanism. Setting a height somewhere also sets the height of similar elements (for `o.key`) on other pages (`o.id`).

I just ran into a user issue due to it:
- open a broadcast with few boards, like https://lichess.org/broadcast/fide-womens-world-chess-championship-2025/shanghai--game-5/xeyMoz7D
- try out the boards list resizer, it won't move much in this case, but it sets a new default to 48px as there is one board
- browse to another broadcast with more boards, like https://lichess.org/broadcast/25th-european-womens-chess-championship-2025/round-8/jGnDcYrv
the boards list uses the default value that was set to 48px. It only shows one board and it's not obvious that there are many more.

Reply:
maybe only chat size needs a generic fallback. good catch and you're right, we should track game size individually with no generic fallback, instead always default to min(fit-content, initialHeight / 2).